src/device/giolister.cpp

```diff
@@ -25,6 +25,9 @@
 #include <glib.h>
 #include <glib-object.h>
 #include <gio/gio.h>
+#ifdef HAVE_GIO_UNIX
+#include <gio/gunixmounts.h>
```

Comment:
`#  include <gio/gunixmounts.h>`

Reply:
I assumed you meant two whitespaces here as well

Reply:
Yes